src/ui/public/chrome/api/nav.js

```diff
@@ -1,11 +1,17 @@
 import { parse, format } from 'url';
-import { startsWith, isString, find } from 'lodash';
+import { find, startsWith, isString, pick } from 'lodash';
 
 export default function (chrome, internals) {
   chrome.getNavLinks = function () {
     return internals.nav;
   };
 
+  chrome.getActiveAppId = function () {
+    const activeAppObj = find(internals.nav, { active: true });
+    const dirtyUrl = activeAppObj.url.toLowerCase().split('kibana#/')[1];
```

Comment:
`kibana#`? Is this meant to only be used by the kibana app? This is part of the chrome, it feels like something would break if we were in an appother than Kibana